specification/maps/data-plane/Route/stable/2025-01-01/route.json

```diff
@@ -0,0 +1,3783 @@
+        },
+        "avoid": {
+          "description": "Specifies restrictions that the route calculation should honor when determining the route. Avoid supports multiple values in a request and is only supported for the driving and truck travelMode.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RouteMatrixAvoidEnum"
+          }
+        },
+        "vehicleSpec": {
+          "$ref": "#/definitions/RouteMatrixVehicleSpec"
+        }
+      }
+    },
+    "RouteMatrixResponse": {
+      "description": "This object is returned from a successful call.",
+      "type": "object",
+      "required": [
+        "type",
+        "geometry",
+        "properties"
+      ],
+      "properties": {
+        "type": {
+          "$ref": "#/definitions/FeaturesItemTypeEnum"
+        },
+        "geometry": {
+          "description": "The geometry object is null",
+          "type": "object"
+        },
+        "properties": {
+          "$ref": "#/definitions/RouteMatrixFeatureProperties"
+        }
+      }
+    },
+    "RouteOperationResponse": {
+      "type": "object",
+      "description": "This object is returned from a successful call.",
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/RouteOperationKindEnum"
+        }
+      },
+      "discriminator": "kind",
+      "required": [
+        "kind"
+      ]
+    },
+    "RouteMatrixAsyncResponse": {
+      "type": "object",
+      "x-ms-discriminator-value": "RouteMatrix",
+      "description": "Specifies the driving instructions and additional properties for each maneuver point in the route Leg.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/RouteOperationResponse"
+        },
+        {
+          "$ref": "#/definitions/RouteMatrixResponse"
+        }
+      ]
+    },
+    "FeatureCollection": {
+      "description": "`GeoJSON` `FeatureCollection` object that contains a list of Features. For more information, see [RFC 7946, section 3.3](https://www.rfc-editor.org/rfc/rfc7946#section-3.3).",
+      "type": "object",
+      "properties": {
+        "type": {
+          "$ref": "#/definitions/FeatureTypeEnum"
+        },
+        "features": {
+          "$ref": "#/definitions/Features"
+        }
+      }
+    },
+    "RegionTravelSummary": {
+      "description": "Travel summary of distance, time, and toll road distance by two entity types: country (e.g. US, Canada) and administrative division or subregion (e.g. “state” in US and “province” in Canada).",
+      "type": "object",
+      "properties": {
+        "countryRegion": {
+          "type": "object",
+          "properties": {
+            "name": {
+              "description": "name of country/region",
+              "type": "string"
+            }
+          }
+        },
+        "adminDistricts": {
+          "description": "adminDistrict array.",
+          "type": "array",
+          "readOnly": true,
```

Comment:
This is one example of _many_ properties in _response-only schemas_ that are marked `readOnly: true`. This has _no meaning_ in a schema that is not sent in a request. These should be removed wherever possible. This does not have to be done in this API version but doing so will avoid a trip through the Breaking Change review board if the fix is done after GA.